DOCSP-41964: Time series collections (#138)
mongodb/docs-php-library: source/databases-collections/time-series.txt

.. _php-time-series:

=======================
Time Series Collections
=======================

.. contents:: On this page
   :local:
   :backlinks: none
   :depth: 1
   :class: singlecol

.. facet::
   :name: genre
   :values: reference
 
.. meta::
   :keywords: chronological, data format, code example

Overview
--------

In this guide, you can learn how to use the {+php-library+} to create
and interact with **time series collections**. These collections store
time series data, which is composed of the following components:

- Measured quantity
- Timestamp for the measurement
- Metadata that describes the measurement

> The following table describes sample situations for which you could store time
> series data:

Reviewer: [s] Can potentially reword if you want to avoid the subjunctive mood:
```suggestion
The following table describes sample situations where you can store time
series data:
```
